docs/source/release/v6.4.0/direct_geometry.rst

```diff
@@ -45,8 +45,8 @@ CrystalField
 New Features
 ############
 
-- The :ref:`Crystal Field Python interface <Crystal Field Python Interface>` has been extended to include functions to calculate the x y and z components of
-  the dipole transition maxtrix, ``getXDipoleMatrixComponent()``, ``getYDipoleMatrixComponent()`` and ``getZDipoleMatrixComponent()``.
+- The :ref:`Crystal Field Python interface <Crystal Field Python Interface>` has been extended to include functions to calculate the x, y, and z components of
+  the dipole transition matrix. ``getXDipoleMatrixComponent()``, ``getYDipoleMatrixComponent()`` and ``getZDipoleMatrixComponent()``.
```

Comment:
This doesn't make a lot of sense to me. Should the fullstop after matrix be a comma instead? If not then the final sentence needs reworking.